apps/web/package.json

```diff
@@ -94,13 +94,13 @@
     "workbox-webpack-plugin": "^6.4.1"
   },
   "scripts": {
-    "---turborepo scripts---": "",
-    "build": "node scripts/build.js",
+    "dev": "npm run prestart && node scripts/start.js",
+    "build": "npm run prestart && node scripts/build.js",
     "deploy": "echo deploy web",
-    "dev": "node scripts/start.js",
     "test": "node scripts/test.js src --watchAll=false",
-    "lint": "eslint src --fix",
-    "clean": "rm -rf .turbo && rm -rf node_modules && rm -rf build"
+    "clean": "rm -rf .turbo && rm -rf node_modules && rm -rf build",
+    "build:tailwind": "tailwindcss build -i src/tailwind.css -o src/tailwind.output.css",
+    "prestart": "npm run build:tailwind"
```

Comment:
Just get rid of this prestart script

dev: npm run build:tailwind && node scripts/start.js
build: npm run build:tailwind && node scripts/build.js

Reply:
Okay, I'll try that, thank you sir!

Reply:
No no just get rid of the prestart script!

You need the others!

dev: npm run build:tailwind && node scripts/start.js
build: npm run build:tailwind && node scripts/build.js

Reply:
Instead of doing npm run prestart && ... you can just do npm run build:tailwind instead, you were like calling another npm script that calls another npm script when you can just call it directly.

Reply:
Fixed